tests/unitarize_link_test.cpp

```diff
@@ -36,7 +36,7 @@ static double unitarize_eps  = 1e-6;
 static bool reunit_allow_svd = true;
 static bool reunit_svd_only  = false;
 static double svd_rel_error  = 1e-4;
-static double svd_abs_error  = 1e-5;
+static double svd_abs_error  = 1e-4;
```

Comment:
Ok, I probably was confused by the error name while this is rather a threshold / tolerance. Anyway, fine with that.